sdk/core/System.ClientModel/src/Internal/ReadTimeoutStream.cs

```diff
@@ -90,7 +89,7 @@ public override int Read(byte[] buffer, int offset, int count)
 
     public override async Task<int> ReadAsync(byte[] buffer, int offset, int count, CancellationToken cancellationToken)
     {
-        var source = StartTimeout(cancellationToken, out bool dispose);
+        CancellationTokenSource source = StartTimeout(cancellationToken, out bool dispose);
```

Comment:
```suggestion
        using CancellationTokenSource source = StartTimeout(cancellationToken, out bool dispose);
```

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/41932